packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/FloatingLink/CustomFloatingLink.tsx

```diff
@@ -0,0 +1,26 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { TEditableProps, useFloatingLinkSelectors } from '@udecode/plate';
+import React from 'react';
+import { EditModal } from './EditLinkModal';
+import { FloatingLink } from './FloatingLink';
```

Comment:
```suggestion
import { EditModal, FloatingLink } from './';
```

Reply:
Not possible, because the folder is named differently